.github/workflows/linters.yml

```diff
@@ -17,36 +17,7 @@ jobs:
       - name: Setup Lighthouse
         run: npm install -g @lhci/cli@0.7.x
       - name: Lighthouse Report
-        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
-  webhint:
-    name: Webhint
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup Webhint
-        run: |
-          npm install --save-dev hint@7.x
-          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.hintrc
-      - name: Webhint Report
-        run: npx hint .
-  stylelint:
-    name: Stylelint
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup Stylelint
-        run: |
-          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
-          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.stylelintrc.json
-      - name: Stylelint Report
-        run: npx stylelint "**/*.{css,scss}"
-        
```

Comment:
- [ ] Please ensure that [html&css + javascript](https://github.com/microverseinc/linters-config/blob/master/html-css-js) are properly set up in your project since this project requires you to use both JavaScript and HTML & CSS :100:
- [ ] Kindly add the parts of the `linters.yml` file you removed and ensure that when you push to github you have `Webhint`, `Lighthouse`and `Stylelint` checks as well. Since these check HTML and CSS respectively :100: 
- Your Current Checks
![image](https://user-images.githubusercontent.com/102699958/228322641-f87927ae-56d1-4097-87e2-e05cb481c960.png)
- What you should have
![image](https://user-images.githubusercontent.com/102699958/228322746-9a41dcb2-9576-441f-b876-813e985f3328.png)